enginerpc/examples/README-gRPC.md

```diff
@@ -4,20 +4,20 @@
 
 Verify that you have installed theses packages on your system else install them. On Centos 7 you can execute this command below :
 
-`yum install python3-devel` </br>
-`yum install gcc-c++` </br>
-`yum install gcc` </br>
-`yum install epel-release` </br>
-`yum install net-tools` </br>
-`yum install wget`
+```yum install python3-devel``` 
+```yum install gcc-c++```
+```yum install gcc```
+```yum install epel-release```
+```yum install net-tools```
+```yum install wget```
```

Comment:
```suggestion
```shell
yum install python3-devel
yum install gcc-c++
yum install gcc
yum install epel-release
yum install net-tools
yum install wget
```
```

Reply:
or use 4 spaces instead of the backticks, like that:
```
    ```
    yum install python3-devel
    yum install gcc-c++
    yum install gcc
    yum install epel-release
    yum install net-tools
    yum install wget
    ```
```